utils/class-users-query-utils.php

```diff
@@ -128,6 +128,9 @@ private static function build_network_capability_where_clause( $capabilities, $r
 		// Build capability conditions - convert capabilities to roles
 		if ( ! empty( $capabilities ) ) {
 			foreach ( $capabilities as $capability ) {
+				$capability_escaped  = esc_sql( $capability );
+				$capability_checks[] = "meta_value LIKE '%\"{$capability_escaped}\";b:1%'";
```

Comment:
To match the exact `b:1;` boolean:
```suggestion
				$capability_checks[] = "meta_value LIKE '%\"{$capability_escaped}\";b:1;%'";
```